devcontainer-admin/.devcontainer.json

```diff
@@ -20,12 +20,6 @@
         "python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
         // "python.linting.pylintPath": "/usr/local/share/pip-global/bin/pylint"
     },
-    "features": {
-        "docker-from-docker": {
-            "version": "latest",
-            "moby": true
-        }
-    },
```

Comment:
This installs the moby-released version of the docker cli, instead of the docker-released one. This is a vestigial dependency imported from GC Notify and is only necessary because they use uncommon/advanced container orchestration techniques that we almost certainly won't.

I experienced a curl failure trying to fetch the moby release, I'm sure it's intermittent, but I'm just going to zap this for now, we probably won't need to bring it back.